newton/tests/test_mujoco_solver.py

```
@@ -4030,6 +4028,240 @@ def test_custom_attributes_from_usd(self):
+        model = builder.finalize()
+        solver = SolverMuJoCo(model, separate_worlds=False)
+
+        assert hasattr(model, "mujoco")
+        assert hasattr(model.mujoco, "dof_ref")
+        assert np.allclose(model.mujoco.dof_ref.numpy(), [45, 0.5, 0])
+        assert np.allclose(solver.mj_model.qpos0, [np.deg2rad(45), 0.5, 0], atol=1e-4)
+
+    @unittest.skipUnless(USD_AVAILABLE, "Requires usd-core")
+    def test_dof_ref_from_usd(self):
+        """Test dof_ref attribute parsing from USD."""
+        from pxr import Sdf, Usd, UsdGeom, UsdPhysics  # noqa: PLC0415
```

Ruff (0.14.10)

Unused `noqa` directive (non-enabled: `PLC0415`)

Remove unused `noqa` directive

(RUF100)